build.sbt

```diff
@@ -10,7 +10,18 @@ import sbt.nio.Keys._
 lazy val scala212 = "2.12.8"
 lazy val scala211 = "2.11.12"
 
-lazy val sparkVersion = sys.env.getOrElse("SPARK_VERSION", "3.0.0")
+lazy val spark2 = "2.4.5"
+lazy val spark3 = "3.0.0"
+
+lazy val sparkVersion = settingKey[String]("sparkVersion")
+ThisBuild / sparkVersion := sys.env.getOrElse("SPARK_VERSION", spark3)
+
+def majorVersion(version: String): String = {
+  StringUtils.ordinalIndexOf(version, ".", 1) match {
```

Comment:
I think it'd be simpler to just have something along the lines of: `version.split('.').head`